config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -451,15 +451,14 @@ presubmits:
         - --provider=gce
         - --tear-down-previous
         - --env=CL2_ENABLE_DNS_PROGRAMMING=true
-        - --env=CL2_ENABLE_VIOLATIONS_FOR_SCHEDULING_THROUGHPUT=false
```

Comment:
I think we need to understand why it was disabled here. 

Setting CL2_ENABLE_DENSITY_TEST=true will enable violations here, which may not be allowed for some reason.

Reply:
Looks like it was turned off, because on 1.18 and 1.19 SchedulingThroughput was below 100. Right now on master, it's much higher than 100, but it's due to increased master size from 4 cores to 8 cores.